source/ios/AdaptiveCards/ADCIOSVisualizer/resources/sample.json

```diff
@@ -50,23 +50,23 @@
 					"subtle": "#6f6f6f"
```

Comment:
why is this sample in the ADCIOSVisualizer project? Is this the same file as the one located in the HostConfigs folder in the samples? do both samples have the same colors?

Reply:
@almedina-ms, iOS visualizer has its own host configs.